mern/server/utils/parsetranscript.js

```diff
@@ -0,0 +1,109 @@
+import QA from "../models/question_answer.js"
+
+// @Params: Transcript: JSON object for transcript with format specified in client utils folder
+
+//First create a lst tree of all the ids of the QA blocks
+//Then based on the list tree, add the QA to the appropriate children of other QA
+
+export default function parseTranscript(transcript){
+
+    // Do breadth first search
+    // On each layer have a counter that inrecements by 1
+    // Each Layer creates QA cards that reference the child
+    const lst = []
+
+
+    for (var i = 0; i<transcript.questions.length; i++){
+        // Creates all the necessary nards
+        // lst = dfs(transcript.questions[i], 0)
+        console.log(transcript.questions[i].q1.intents[0].intent1.children)
```

Comment:
What is the reason we have a parse.js file and a parsetranscript.js file? I guess a better way to phrase my question is, in what case do we use this one instead of parse.js? 

Some questions about this function as well: What does it mean by "do breadth first search"? Also how are you accessing q1 or intent1 for example... (Sorry I have so many questions 😭) I think when you get the opportunity, it'd be great if you could show us how this function would parse an example transcript during our meetings.

Reply:
Yea, parsetranscript.js was made for the old transcript which won't be used. This file is now deleted.